src/main/java/com/worldwarofants/game/module/newgame/NewGameView.java

```diff
@@ -10,11 +10,13 @@ public NewGameView(NewGameViewModel viewModel) {
 	}
 
 	public void renderNewGameScreen() {
+		String startGamePrompt = String.format("* Start Game - %s ", NewGameCommandHandler.COMMAND_START_NEW_GAME);
+		String backPrompt = String.format("* Back - %s ", NewGameCommandHandler.COMMAND_BACK);
```

Comment:
This whole way of printing the info could be better. `* Back - back` doesn't really mean anything. I'll give an example with Git again. When you enter `help`, you see the name of the command and a description of it like this http://xahlee.info/linux/i/git_help_screen.png